app/renderer/js/components/tab.ts

```diff
@@ -28,7 +28,12 @@ export default class Tab extends BaseComponent {
 	}
 
 	activate(): void {
-		this.$el.classList.add('active');
+		if (this.webview.$el) {
+			if (this.webview.$el.classList.contains('disabled')) {
+				this.webview.$el.classList.remove('disabled');
+			}
+			this.webview.$el.classList.add('active');
+		}
```

Comment:
This should already happen inside `this.webview.load()`, right?

Reply:
Yes. I have to use `this.webview.load()` instead of rewriting a new function.